tests/src/FunctionalJavascript/ThunderJavascriptTestBase.php

```diff
@@ -453,24 +453,38 @@ public function runCron() {
   }
 
   /**
-   * Click article save option based on index of action.
+   * Click article save.
+   */
+  protected function clickSave() {
+    $page = $this->getSession()->getPage();
+
+    $page->find('xpath', '//div[@data-drupal-selector="edit-actions"]/input[@id="edit-submit"]')
+      ->click();
+  }
+
+  /**
+   * Set entity status.
    *
-   * 1 - Save as unpublished (default).
-   * 2 - Save and publish.
+   * TRUE - Unpublished.
```

Comment:
Comment should be adjusted - it's a bit confusing.

And maybe it would be good to rename method, so it's clear that Published status will be set - maybe: `setPublishedStatus` or something like that.